csp_billing_adapter/exceptions.py

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Great :tada: , I had a note to create native exceptions but forgot to add it as an issue.